mapper.go

```diff
@@ -24,7 +24,7 @@ import (
 )
 
 var (
-	metricLineRE      = regexp.MustCompile(`^(\*\.|[^*.]+\.)*(\*|[^*.]+)$`)
+	metricLineRE      = regexp.MustCompile(`^(\*\.|[^*.]+\.|\.)*(\*|[^*.]+)$`)
```

Comment:
Please see line 285 in

http://codenav.org/code.html?project=/com/yammer/metrics/metrics-graphite/2.2.0&path=/Source%20Packages/com.yammer.metrics.reporting/GraphiteReporter.java